presto/dataset.py

```diff
@@ -163,6 +174,17 @@ def __getitem__(self, idx):
         )
 
 
+def filter_remove_noncrops(df: pd.DataFrame) -> pd.DataFrame:
+    noncrop_labels = [10, 11, 12, 13]
```

Comment:
I'm confused. `[10, 11, 12, 13]` should be considered `crop_labels`, and in a croptype task, we should filter the `df` to only keep the rows which have a `LANDCOVER_LABEL` matching values in this list. It seems the opposite is happening here and we're throwing away all crops? I'd think actually all maize labels would be gone now. Or am I misreading this part?

Reply:
🤦 Thanks for catching this